clients/python/README.md

```diff
@@ -3,7 +3,9 @@
 - Python 3
 
 ## Setup
-`pip install -r requirements.txt`
+1. `pip install -r requirements.txt`
+2. Rename `player.default.json` to `player.json` and replace the
+   `team_secret` by the value that was given to you by the organiser
 
 ## Joining the Ranked Game
 `python play.py`
```

Comment:
Hmm but I don't think we should write "in `clients/python`" because the participants will only get this folder, they won't have a hierarchy of folders like we do. :P I added `./` to make it implicit that it's in the current folder (so they have to be *in that folder*). Is that okay with you? How else would you phrase it?